clients/cobol-lsp-vscode-extension/src/__tests__/JavaCheckTest.ts

```diff
@@ -12,10 +12,11 @@
  *   Broadcom, Inc. - initial API and implementation
  */
 
-import { JavaCheck } from "../services/JavaCheck";
+import * as cp from "child_process";
+import {JavaCheck} from "../services/JavaCheck";
 
-describe("Check Java version", () => {
-    it("checks if Java version is supported", async () => {
+describe("Checks Java version", () => {
+    it("when version is supported", async () => {
```

Comment:
it when version is supported - not really readable )

Reply:
changed, thanks